feg/cloud/go/services/feg_relay/servicers/feg_to_gw_relay.go

```diff
@@ -142,3 +141,20 @@ func getFegServedIds(networkId string) ([]string, error) {
 	}
 	return networkFegConfigs.ServedNetworkIds, nil
 }
+
+func validateFegContext(ctx context.Context) error {
+	fegId := protos.GetClientGateway(ctx)
```

Comment:
`fegId`->`fegID`?

Reply:
Note we have `fegId` across all the system in feg. There is no instances of FegID though

I think the go conventions is to camelcase strict, no matter if it is an acronym. Teid better than TEID

Reply:
ooo got it :) So maybe the convention is slightly different between orc8r and feg.